javascript/javascript3/week1/myOwnJsonFile.json

```diff
@@ -0,0 +1,31 @@
+[
```

Comment:
Looks good to me. Maybe you could add a nested array or something, to make it slightly more challenging.